blazor/rich-text-editor/form-validation.md

```diff
@@ -13,64 +13,19 @@ This following sample demonstrate how to get the Rich Text Editor validation err
 
```

Comment:
use this content for the above line - The Rich Text Editor support validation using the [EditForm]( https://docs.microsoft.com/en-us/aspnet/core/blazor/forms-validation?view=aspnetcore-6.0). The user type text content inside the editor is validated with [data annotations]( https://docs.microsoft.com/en-us/aspnet/core/mvc/models/validation?view=aspnetcore-5.0) attributes. In the following example, the `FormModel` class has the `Description` property marked required with the `RequiredAttribute` and `MinLengthAttribute` minimum string length validation and error message. The Description property binding to editor `@bind-Value` property and validation work based on user input.